crates/telio-wg/src/pq.rs

```diff
@@ -16,19 +16,21 @@ use pnet_packet::{
 };
 use pqcrypto_kyber::kyber768;
 use pqcrypto_traits::kem::{Ciphertext, PublicKey, SharedSecret};
-use rand::{prelude::Distribution, SeedableRng};
+use rand::{prelude::Distribution, RngCore, SeedableRng};
 use tokio::net::{ToSocketAddrs, UdpSocket};
 
 const SERVICE_PORT: u16 = 6480;
 const LOCAL_PORT_RANGE: RangeInclusive<u16> = 49152..=u16::MAX; // dynamic port range
 const LOCAL_IP: Ipv4Addr = Ipv4Addr::new(10, 5, 0, 2);
 const REMOTE_IP: Ipv4Addr = Ipv4Addr::new(10, 5, 0, 1);
-const HANDSHAKE_VERSION: u32 = 1;
+const PQ_PROTO_VERSION: u32 = 1;
 const CIPHERTEXT_LEN: u32 = kyber768::ciphertext_bytes() as _;
 
 const IPV4_HEADER_LEN: usize = 20;
 const UDP_HEADER_LEN: usize = 8;
 
+type RekeyStatus = u8;
```

Comment:
Yeah, it's an internal (private) helper. And to be honest I don't know how many different values it takes, because it's the u8 field in the response packet. Will leave it as is